### ISSUE TITLE
Material radio component traps focus

### ISSUE THREAD

**Issue by tokewf** [ngx-formly/ngx-formly#1691]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
```

**Current behavior**
The implementation of Material radio button/radio group traps focus within the form control. This happens because the containing element passes on the focus to the firstmost radio option in the group. Line 32 in src/material/radio/src/radio.type.ts:

```
onContainerClick(event: MouseEvent): void {
  if (this.radioGroup._radios.length && !this.radioGroup.selected) {
    this.radioGroup._radios.first.focus();
  }
  super.onContainerClick(event);
}
```
The focus does not update correctly until an option has been selected two times. This can be the same option or two different options. I suspect that the line `this.radioGroup._radios.first.focus()` or indeed the condition for the `if` statement above is the culprit.

The problem with this behavior is that when the user fills out a form, selects a radio option and moves on to the following field(s), the radio group remains in focus—i.e. keeps its 'active'—styling even though focus has moved to another field.

**Expected behavior**
The focus of the radio group should be correctly tracked. Further, I don't see any particular reason why focus should be automatically moved to the firstmost radio option, as this is usually handled simply by pressing the Tab key an extra time.

**Minimal reproduction of the problem with instructions**
1. Click anywhere within the containing element of a radio group. The firstmost radio option receives focus
2. Now click anywhere outside this containing element. The entire form group should lose focus (i.e. lose 'active' styling), but does not
3. Select an option in the radio group
4. Click outside the containing element. Still no focus change.
5. Select another (or the same) option in the radio group
6. Click outside the containing element. Focus now correctly updates.

StackBlitz: https://stackblitz.com/edit/ngx-formly-ui-material-h8udre?file=app/app.component.ts

**Please tell us about your environment:**
macOS 10.14.5
Firefox 68.0.1
Angular 7.0.3
ngx-formly 5.3.0
rxjs ^6.2.2

**Comment by tokewf** [comment]
The same seems to apply for `multicheckbox`. Focus is kept even after clicking outside the containing element.

**Comment by aitboudad** [comment]
checked on my side and it seems the previous fix also fixed the `multicheckbox` issue too (2 in 1 😄).

**Comment by tokewf** [comment]
@aitboudad that's wonderful :) I am on version 5.3.0, how can I get a hold of your fix?

**Comment by aitboudad** [comment]
the fix should be landed in `v5.4`, I'll try my best to make it available next Monday.
if you can't wait until then, let me know to provide you a workaround.

**Comment by tokewf** [comment]
Next Monday would be brilliant, no need for a workaround.

**Comment by aitboudad** [comment]
released in `5.4` version.
